src/platform/icelake/include/platform/alh.h

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
include/platform/drivers/alh.h

Reply:
This is an icelake specific header

Reply:
Yes, exactly what I meant: src/platform/icelake/include/platform/drivers/alh.h (see latest dir structure for reference).